entity-service-client/src/main/java/org/hypertrace/entity/data/service/client/EdsCacheKey.java

```diff
@@ -14,4 +16,18 @@ public EdsCacheKey(String tenantId, String entityId) {
   public String toString() {
     return String.format("__%s__%s", this.tenantId, this.entityId);
   }
+
+  @Override
+  public int hashCode() {
```

Comment:
oops.. So, was this making the entire caches ineffective?

Reply:
Yes :)